### PR TITLE
Move security-related HTTP response headers from Rust to nginx.conf

### PR DESCRIPTION
**Pull request by kzys**
We'd like to have these headers on the FastBoot server as well.

### CODE REVIEW

**Comment by kzys** [comment]
Regarding `Content-Security-Policy`, we could either 1) implement a similar logic on nginx.conf.erb or 2) introduce a new environment variable just for this purpose.

**Comment by kzys** [comment]
I'm going to implement the URL construction logic by reading `S3_CDN`, `S3_BUCKET` and `S3_REGION` from nginx.conf.erb. Please let me know we do want to avoid the duplication of the logic!

File: config/nginx.conf.erb [hunk 1]
```diff
@@ -121,6 +139,11 @@ http {
 			expires max;
 		}
 
+		add_header X-Content-Type-Options "no-sniff";
```
[the remark this answers is not in the document]
Comment (kzys) [reply config/nginx.conf.erb]: Oh, good catch! Fixed.

**Comment by jtgeibel** [comment]
LGTM!

@bors r+

**Pull request merged**